packages/ui/src/services/current-app.ts

```diff
@@ -0,0 +1,15 @@
+import type { App } from 'vue'
+import { inject as vueInject } from 'vue'
+
+/**
+ * Similar to `getCurrentInstance` but for plugins, so we can use inject in plugins.
+ */
+let app: App | null
+
+export const setCurrentApp = (instance: App | null) => { app = instance }
+export const getCurrentApp = () => app
+
+/** Wrapper around vue inject, so it can be used in plugins */
+export const inject = ((key: string, value?: any) => {
```

Comment:
I think this will be confusing for us. Maybe will be better to name like `injectPlugin` and leave vue's inject as is?

Reply:
I'll refactor this later. Not sure for now how to use this yet.